requirements.txt

```diff
@@ -8,7 +8,8 @@ Jinja2>=2.7.3
 Sphinx>=1.2.3
 docopt>=0.6.2
 simplejson>=3.8.1
+requests>=2.9.1
 
 # Python3 compat
 future>=0.15.2
-configparser>=3.3.0; python_version < '3'
+#configparser>=3.3.0; python_version < '3'
```

Comment:
configparser is required and should not be commented for python2.
This issus is  "python_version < '3' " that need a really recent version of python setuptools.
I think that the current way to manage that is maybe to make a quick patch with sed in the .spec file.
So remove that "python_version < '3' " for python2 pkg and remove the complete line for python3 pkg.